src/environment.js

```diff
@@ -18,3 +18,4 @@ export const chargebeeSubdomain = () => {
 };
 
 export const siteRecaptchaKey = () => process.env.GATSBY_SITE_RECAPTCHA_KEY;
+export const siteRecaptchaInvisibleKey = () => process.env.GATSBY_SITE_RECAPTCHA_INIVISIBLE_KEY;
```

Comment:
Does this actually work? I thought Netlify didn't support it.

Reply:
I ended up not using this, need to remove it. Thanks!